org.civicrm.reqangular/src/common/angular-date/src/services/DateFormatFactory.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * This factory provides access to date format from CRM settings
+ * @param $q
+ * @returns {{dateFormat: null, getDateFormat: getDateFormat}}
+ * @constructor
+ */
+function DateFormatFactory($q) {
```

Comment:
Why is this service called a `Factory`? This doesn't seem to match the definition of a factory, it just returns a string